src/Sentry/Protocol/Envelopes/EnvelopeItem.cs

```diff
@@ -417,18 +403,6 @@ private static async Task<ISerializable> DeserializePayloadAsync(
             return new JsonSerializable(sentryEvent);
         }
 
-        // User report
-        if (string.Equals(payloadType, TypeValueUserReport, StringComparison.OrdinalIgnoreCase))
```

Comment:
question: Feedback

After removing `UserFeedback.FromJson` I believe there is no `SentryFeedback.FromJson` ... is that needed?

Also, I haven't added Structured-Logs to this Deserialization-Flow ... is this something we need to follow-up with?